README.md

```diff
@@ -1522,6 +1528,26 @@ There are two placeholders that can be used in the conditions for endpoints of t
 - The placeholder `[DNS_RCODE]` resolves to the name associated to the response code returned by the query, such as
 `NOERROR`, `FORMERR`, `SERVFAIL`, `NXDOMAIN`, etc.
 
+### Monitoring an endpoint using SSH command
+You can monitor endpoints using SSH by prefixing `endpoints[].url` with `ssh:\\`:
+```yaml
+endpoints:
+  - name: ssh-example
+    url: "ssh://example.com"
+    ssh:
+      port: "22"
+      username: "root"
+      password: "password"
+      command: "uptime"
```

Comment:
Just thinking out loud here, feel free to disagree.

---

I think the port can be part of the URL:
```yaml
endpoints:
  - name: ssh-example
    url: "ssh://example.com:22"
    ssh:
      username: "root"
      password: "password"
      command: "uptime"
```
I was going to recommend also having the username and password as part of the url, i.e. `ssh://username:password@example.com:22` & use the body as the command, but I'm not 100% sure I'd feel safe having the username and password in the URL, especially considering the hostname is displayed on the UI, so I'm okay with giving the username and password their own fields.

That being said, the `command` feels weird to me, I think providing a command should go through the `endpoints[].body` field & the exit code _could_ be reflected through the `[STATUS]` placeholder.

Then again, `[EXIT_CODE]` makes sense, as eventually support for shell commands will be added to Gatus AND we do already have `[DNS_RCODE]` specifically for endpoints of type DNS...